src/Snackbar/Snackbar.js

```diff
@@ -103,6 +103,11 @@ class Snackbar extends Component {
     muiTheme: PropTypes.object.isRequired,
   };
 
+  static reasons = {
```

Comment:
Should this be renamed to `closeReasons`?

Reply:
Not sure that's really needed for something as simple as this PR is addressing?

Reply:
Also matches the name of the argument given in the documentation..

Reply:
http://www.material-ui.com/#/components/snackbar

"The reason parameter can optionally be used to control the response to onRequestClose, for example ignoring clickaway."

Reply:
ok - sounds good